src/services/highcharts-factory/highcharts-factory.svc.coffee

```diff
@@ -106,8 +112,47 @@ angular
           thresholdBar = _.map(new Array(projectionIntervalLength), -> parseFloat(threshold.value))
         serie.data.push.apply(serie.data, thresholdBar)
         # Note: series can only be added after initial render via the `addSeries` method.
-        @hc.addSeries(serie, true)
-      @hc
+        chart.addSeries(serie, true)
+      chart
+
+    # Render a threshold serie dataLabel when KPI triggered
+    renderThresholdTooltip: (chart, options = @options)->
+      thresholdSerie = _.find(chart.series, (s)->
+        s.options.triggered && s.name.toLowerCase() == 'threshold kpi'
+      )
+      return removePointDataLabels(@triggeredPoint) unless thresholdSerie
```

Comment:
Are you sure we should have `@triggeredPoint` as a global variable? shouldn't it remain in the method's scope?